astropy/utils/console.py

```diff
@@ -813,7 +815,8 @@ def work(i):
                     if (i % chunksize) == 0:
                         bar.update(i)
             else:
-                p = multiprocessing.Pool()
+                p = multiprocessing.Pool(
+                    processes=(multiprocess if multiprocess > 1 else None))
```

Comment:
Seems counter-intuitive to me that setting `multiprocess` to 1 would use up *all* the cores. I propose that any invalid integer value to re-route to the "no multiprocessing" logic block. What do you think?

Reply:
what about using `-1`, like in `Table.pprint`?